Cargo.toml

```diff
@@ -17,8 +17,8 @@ rev = "0c0cae6fa22b751e7fd95d7ce7556ac6d7b7010e"
 
 [workspace.dependencies.soroban-spec]
 version = "0.8.5"
```

Comment:
This needs changing to 0.8.6.

Reply:
Do we still need all the rev's and git in all these deps?

Reply:
not sure, but let's defer that discussion so we can release it ?